src/hooks/users/useConnectWorldUsers.ts

```diff
@@ -1,29 +1,14 @@
-import { useFirestoreConnect } from "hooks/useFirestoreConnect";
-import { useSovereignVenueId } from "hooks/useSovereignVenueId";
-import { useVenueId } from "hooks/useVenueId";
+import { useEffect } from "react";
 
-export const useConnectWorldUsers = () => {
-  const venueId = useVenueId();
-
-  const { sovereignVenueId, isSovereignVenueIdLoading } = useSovereignVenueId({
-    venueId,
-  });
-
-  useFirestoreConnect(() => {
-    if (isSovereignVenueIdLoading || !sovereignVenueId || !venueId) return [];
+import { useWorldUsersContext } from "./useWorldUsers";
 
-    const relatedLocationIds = [venueId];
-
-    if (sovereignVenueId) {
-      relatedLocationIds.push(sovereignVenueId);
-    }
-
-    return [
-      {
-        collection: "users",
-        where: ["enteredVenueIds", "array-contains-any", relatedLocationIds],
-        storeAs: "worldUsers",
-      },
-    ];
-  });
+export const useConnectWorldUsers = () => {
+  // We mostly use this here to ensure that the WorldUsersProvider has definitely been connected
+  useWorldUsersContext();
+
+  useEffect(() => {
+    console.log(
+      "useConnectWorldUsers is practically a noop at the moment, all the real work happens in WorldUsersProvider in src/hooks/users/useWorldUsers.tsx"
+    );
+  }, []);
```

Comment:
It's only used in 2 places. Doesn't it make sense just to use context there?
![image](https://user-images.githubusercontent.com/63194656/124873961-d0d86900-dfcf-11eb-9c13-af7d6651ee9f.png)

Reply:
Probably. This was mostly a placeholder as I was working through this + before I was sure what was going to actually be needed still.

Reply:
fixed in 985301f16c477e29959e12a6301f71a8901cc5d2